tests/test_enterprise/api/test_views.py

```diff
@@ -663,6 +669,7 @@ def test_enterprise_customer_courses(
         """
         Make sure the enterprise courses view returns correct data.
         """
+        self.set_jwt_cookie(ENTERPRISE_ADMIN_ROLE, enterprise_uuid)
```

Comment:
why not just globally set the role to the operator in setup so we don't have to manually do this in each test? would that work?